cishouseholds/pipeline/pipeline_stages.py

```diff
@@ -624,23 +625,51 @@ def lookup_based_editing(
         input file path name for travel_countries corrections lookup file
     edited_table
     """
-
     df = extract_from_table(input_table)
-    cohort_lookup = extract_from_table(cohort_lookup_table)
-    travel_countries_lookup = extract_from_table(travel_countries_lookup_table)
-    tenure_group = extract_from_table(tenure_group_table).select(
-        "UAC", "numAdult", "numChild", "dvhsize", "tenure_group"
-    )
-    for lookup_table_name, lookup_df, join_on_column_list in zip(
-        [cohort_lookup_table, travel_countries_lookup_table, tenure_group_table],
-        [cohort_lookup, travel_countries_lookup, tenure_group],
-        [["participant_id", "old_cohort"], ["been_outside_uk_last_country_old"], ["UAC"]],
-    ):
-        check_lookup_table_joined_columns_unique(
-            df=lookup_df, join_column_list=join_on_column_list, name_of_df=lookup_table_name
-        )
 
-    df = transform_from_lookups(df, cohort_lookup, travel_countries_lookup, tenure_group)
```

Comment:
I'd keep the refactored code under the `transform_from_lookups` function, as this is tested (see failing test here)

Reply:
The code is still in the stage, rather than the higher level function it was in before